src/test/java/ch/ivyteam/ivy/maven/compile/TestCompileProjectMojo.java

```diff
@@ -36,63 +36,60 @@
   public CompileMojoRule<CompileProjectMojo> compile = new CompileMojoRule<CompileProjectMojo>(CompileProjectMojo.GOAL)
   {
     @Override
-    protected void before() throws Throwable 
+    protected void before() throws Throwable
     {
       super.before();
       // use same project as first rule/mojo
       testMojo = (CompileTestProjectMojo) lookupConfiguredMojo(project, CompileTestProjectMojo.GOAL);
       configureMojo(testMojo);
     }
   };
-  
+
   @Test
   public void buildWithExistingProject() throws Exception
   {
     CompileProjectMojo mojo = compile.getMojo();
-    
+
     File dataClassDir = new File(mojo.project.getBasedir(), "src_dataClasses");
     File wsProcDir = new File(mojo.project.getBasedir(), "src_wsproc");
     File classDir = new File(mojo.project.getBasedir(), "classes");
     FileUtils.cleanDirectory(wsProcDir);
     FileUtils.cleanDirectory(dataClassDir);
-    
+
     mojo.buildApplicationDirectory = Files.createTempDirectory("MyBuildApplication").toFile();
     mojo.execute();
-    
+
     assertThat(findFiles(dataClassDir, "java")).hasSize(2);
     assertThat(findFiles(wsProcDir, "java")).hasSize(1);
-    
+
     assertThat(findFiles(classDir, "txt"))
       .as("classes directory must be cleand by the builder before compilation")
       .isEmpty();
     assertThat(findFiles(classDir, "class"))
       .as("compiled classes must exist. but not contain any test class.")
       .hasSize(4);
-    assertThat(findFiles(classDir, "xml"))
-      .as("resources from the src folder must be copied to the classes folder")
-      .hasSize(1);
```

Comment:
was created to assure that persistence.xml was copied from src ... do we still need that? I don't think so right @alexsuter ? I assume we access the persistence.xml resource without loading it from target/classes dir, right?

Reply:
persistence.xml stays in config folder. we never have used class loading mechanism to load this file. I dont know why so many hacks was implemented.